feature-add-meal/build.gradle

```diff
@@ -0,0 +1,45 @@
+apply plugin: "com.android.library"
+apply from: rootProject.file("common.gradle")
+
+def composeVersion = "1.0.0-alpha07"
+
+dependencies {
+    // Feature module
+    implementation project(":lib-theming")
+    // Plugin
+    implementation "com.vanpra.compose-material-dialogs:datetime:0.2.6"
+    implementation("com.benasher44:uuid:0.2.2")
+    // Compose
+    implementation "androidx.compose.compiler:compiler:$composeVersion"
+    implementation "androidx.compose.ui:ui:$composeVersion"
+    // Tooling support (Previews, etc.)
+    implementation "androidx.ui:ui-tooling:$composeVersion"
+    // Foundation (Border, Background, Box, Image, Scroll, shapes, animations, etc.)
+    implementation "androidx.compose.foundation:foundation:$composeVersion"
+    // Material Design
+    implementation "androidx.compose.material:material:$composeVersion"
+    // Material design icons
+    implementation "androidx.compose.material:material-icons-core:$composeVersion"
+    implementation "androidx.compose.material:material-icons-extended:$composeVersion"
+    // Integration with observables
+    implementation "androidx.compose.runtime:runtime-livedata:$composeVersion"
+    implementation "androidx.compose.runtime:runtime-rxjava2:$composeVersion"
+    implementation "com.google.android.gms:play-services-maps:17.0.0"
+
+    // Tests
+    testImplementation("org.jetbrains.kotlinx:kotlinx-coroutines-test:1.4.2")
+    implementation "junit:junit:4.12"
+}
+
+android {
+    buildFeatures {
+        compose true
+    }
+    composeOptions {
+        kotlinCompilerVersion = "1.4.10"
```

Comment:
Is it worth this being in common.gradle?

Reply:
Do we want to have Jetpack Compose on another feature module? In that case, I am 💯 to get this moving to `common.gradle`